src/globals.h

```diff
@@ -101,6 +101,7 @@ typedef struct {
     void *stack_root;
     apdu_handler handlers[INS_MAX + 1];
     bip32_path_with_curve_t path_with_curve;
+    bool is_blank_screen;
```

Comment:
It makes more sense to me to have this field in the `dynamic_display` field.

Reply:
Done.